ios/docs/Controls/Label.md

```diff
@@ -0,0 +1,33 @@
+# Label
+## Overview
+Use `Label` to standardize text across your app.
+
+The different styles and colors of `Label` are pictured below. These styles and colors can be customized with the `attributedText` and `maxFontSize` properties. If the device's accessibility text size is changed, the `Label` text size will adjust accordingly.
+![Label.png](.attachments/Label.png)
```

Comment:
might be worth giving more context on accessibility text size. potential reference poits of apple hig https://developer.apple.com/design/human-interface-guidelines/foundations/accessibility/#:~:text=see%20Keyboards.-,Text%20display,-In%20iOS%2C%20iPadOS  and https://developer.apple.com/design/human-interface-guidelines/foundations/typography/